x-pack/plugins/security_solution/public/detections/components/alerts_table/alerts_grouping.tsx

```diff
@@ -132,10 +132,6 @@ const GroupedAlertsTableComponent: React.FC<AlertsTableComponentProps> = (props)
     setPageIndex((curr) => curr.map(() => DEFAULT_PAGE_INDEX));
   }, []);
 
-  useEffect(() => {
```

Comment:
this is an unrelated change, but a [follow up from the last PR](https://github.com/elastic/kibana/pull/152862#discussion_r1175678532).